public/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -271,16 +282,14 @@ Optimistic sync is a post-merge synchronization strategy designed to be opt-in a
 
 #### Checkpoint sync {#checkpoint-sync}
 
-Checkpoint sync, also known as weak subjectivity sync, creates a superior user experience for syncing Beacon Node. It's based on assumptions of [weak subjectivity](/developers/docs/consensus-mechanisms/pos/weak-subjectivity/) which enables syncing Beacon Chain from a recent weak subjectivity checkpoint instead of genesis. Checkpoint sync makes the initial sync time significantly faster with similar trust assumptions as syncing from [genesis](/glossary/#genesis-block).
+A checkpoint sync, also known as weak subjectivity sync, creates a superior user experience for syncing a Beacon Node. It's based on assumptions of [weak subjectivity](/developers/docs/consensus-mechanisms/pos/weak-subjectivity/) which enables syncing the Beacon Chain from a recent weak subjectivity checkpoint instead of genesis. Checkpoint syncs make the initial sync time significantly faster with similar trust assumptions as syncing from [genesis](/glossary/#genesis-block).
 
-In practice, this means your node connects to a remote service to download recent finalized states and continues verifying data from that point. Third party providing the data is trusted and should be picked carefully.
+In practice, this means your node connects to a remote service to download recent finalized states and continues verifying data from that point. The third party providing the data is trusted and should be picked carefully.
 
-More on [checkpoint sync](https://notes.ethereum.org/@djrtwo/ws-sync-in-practice)
+More on [checkpoint syncs](https://notes.ethereum.org/@djrtwo/ws-sync-in-practice).
```

Comment:
@m4sterbunny Hah fair enough... I tend to lean towards consistency with these things... we're using "sync" in all of the headers and "More on snap sync", "More on optimistic sync" for the trailing read more links above... I'd suggest we just go with "More on checkpoint sync"

```suggestion
More on [checkpoint sync](https://notes.ethereum.org/@djrtwo/ws-sync-in-practice)
```

Or if you disagree, please at least remove the trailing hard stop:

```suggestion
More on [checkpoint syncs](https://notes.ethereum.org/@djrtwo/ws-sync-in-practice)
```